src/db/migrations/postgres/CreateCoinAndCoinPriceTables1631234567894.ts

```diff
@@ -0,0 +1,28 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateCoinAndCoinPriceTables1631234567894
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      CREATE TABLE coin (
+        id serial PRIMARY KEY,
+        symbol varchar,
+        name varchar,
+        total_supply float
+      )
+    `);
+
+    await queryRunner.query(`
+      CREATE TABLE coin_price (
+        id serial PRIMARY KEY,
+        price float
```

Comment:
You completely deleted coinId, it's not right.
You have to have `coinID` ad number here.
And don't forget to update your entity as well!

Reply:
oh, sorry. edited.